src/lib/logger.js

```diff
@@ -93,6 +93,10 @@ function setup(logs) {
 		},
 	});
 
+       process.on('SIGUSR2', function () {
+               log.reopenFileStreams();
```

Comment:
Check the linting is failing, `log` is not defined.

you can run locally `npm run lint` to double check.

Reply:
Fixed. Lint passes now. Tested on Ubuntu Trusty.